.github/settings.yml

```diff
@@ -35,24 +35,23 @@ collaborators:
   - username: caniszczyk
     permission: admin
 
-  - username: mrbobbytables
-    permission: push
-
-  - username: idvoretskyi
-    permission: admin
+#  - username: idvoretskyi
```

Comment:
Can we keep Ihor on here?

Reply:
Since I know he's not using his account, I wanted to comment it out.  It's effectively a security hole.

After the war, we'll re-activate it.